modules/nw-aws-switching-clb-with-nlb.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nw-aws-switching-clb-with-nlb_{context}"]
+= Switching the Ingress Controller from using a Classic Load Balancer to a Network Load Balancer
+
+You can switch the Ingress Controller that is using a Classic Load Balancer (CLB) to one that uses a Network Load Balancer (NLB) on AWS.
+
+Switching between these load balancers will not delete the IngressController object.
+
+[WARNING]
+====
+This procedure might cause the following issues:
+
+* An outage that can last several minutes due to new DNS records propagation, new load balancers provisioning, and other factors. IP addresses and canonical names of the Ingress Controller load balancer might change after applying this procedure.
+
+* Leaked load balancer resources due to a change in the annotation of the service.
+====
+
+.Procedure
+
+. Modify the existing Ingress Controller that you would like to switch to using an NLB. The following example assumes that your default Ingress Controller has an `External` scope and no other customizations:
++
+.Example `ingresscontroller.yaml` file
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  creationTimestamp: null
+  name: default
+  namespace: openshift-ingress-operator
+spec:
+  endpointPublishingStrategy:
+    loadBalancer:
+      scope: External
+      providerParameters:
+        type: AWS
+        aws:
+          type: NLB
+    type: LoadBalancerService
+----
++
+[NOTE]
+====
+If you do not specify a value for the `spec.endpointPublishingStrategy.loadBalancer.providerParameters.aws.type` field, the Ingress Controller uses the `spec.loadBalancer.platform.aws.type` value from the cluster `Ingress` configuration that was set during installation.
+====
++
+[TIP]
+====
+If your Ingress Controller has other customizations that you want to update, such as changing the domain, consider force replacing the Ingress Controller definition file instead.
+====
+
+. Apply the changes to the Ingress Controller YAML file by running the following command:
```

Comment:
I think you could remove "the following command". I will send you the slack thread I am thinking of. 
```suggestion
. Apply the changes to the Ingress Controller YAML file by running:
```

Reply:
or 
```suggestion
. Apply the changes to the Ingress Controller YAML file by running the command:
```